pkg/steps/run.go

```diff
@@ -105,18 +108,28 @@ type subtestReporter interface {
 	SubTests() []*junit.TestCase
 }
 
-func runStep(ctx context.Context, node *api.StepNode, out chan<- message) {
+func runStep(ctx context.Context, node *api.StepNode, out chan<- Message) {
 	start := time.Now()
 	err := node.Step.Run(ctx)
 	var additionalTests []*junit.TestCase
 	if reporter, ok := node.Step.(subtestReporter); ok {
 		additionalTests = reporter.SubTests()
 	}
```

Comment:
This has no impact on the execution time of the step, so it could be moved outside the timing code to allow us to reuse this function in `cmd/ci-operator/main.go`.

Reply:
Which function do you want to use in `cmd/ci-operator/main.go`?